flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples/wordcount/WordCount.java

```diff
@@ -187,6 +187,16 @@ public static void main(String[] args) throws Exception {
      * Implements the string tokenizer that splits sentences into words as a user-defined
      * FlatMapFunction. The function takes a line (String) and splits it into multiple pairs in the
      * form of "(word,1)" ({@code Tuple2<String, Integer>}).
+     *
+     * <p>The tokenizer performs the following operations:
+     * <ul>
+     *   <li>Converts input text to lowercase for case-insensitive word counting</li>
```

Comment:
I think the code is pretty self explanatory - I suggest that we do not need this extra comment.